### PR TITLE
monitoring: Add sends metrics to influxdb 

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: monitoring/run_monitoring.js [hunk 1]
```diff
@@ -76,6 +98,8 @@ const getConfig = () => {
     return config;
 };
 
+const startTime = +new Date();
```
Comment (fredericosilva) [inline monitoring/run_monitoring.js]:
```suggestion
const startTime = Date.now();
```

File: monitoring/run_monitoring.js [hunk 2]
```diff
@@ -126,7 +150,68 @@ const sendMail = async (config, success, report) => {
     await smtpTransport.close();
 };
 
-const runProcess = async (cmd, env = {}, timeout = 60 * 30) => new Promise((resolve, reject) => {
+const safe = text => `${text}`.replace(/[^a-zA-Z0-9\-]/g, '_');
+
+const sendInflux = (conn, measurement, labels, fields, timestamp) => {
+    const unix = `${timestamp || +new Date() * 1000000}`;
```
Comment (fredericosilva) [inline monitoring/run_monitoring.js]:
```suggestion
    const unix = `${timestamp || Date.now() * 1000000}`;
```

File: monitoring/run_monitoring.js [hunk 3]
```diff
@@ -181,12 +276,20 @@ const main = async () => {
...
+    };
+
     let all_pass = true;
     const runTest = async test_path => {
         let output = `== ${test_path} ==\n`;
         try {
-            output+= await runIsolated(config, `${config.MONITORING_CMD} ${test_path}`);
+            output += await runIsolated(config, `${config.MONITORING_CMD} ${test_path}`, { influxConn });
         } catch (err) {
+            console.log(err);
```
Comment (fredericosilva) [inline monitoring/run_monitoring.js]: is this intended?

Comment (ad-m) [inline monitoring/run_monitoring.js]:
```suggestion
```

File: monitoring/run_monitoring.js [hunk 4]
```diff
@@ -158,21 +243,31 @@ const runProcess = async (cmd, env = {}, timeout = 60 * 30) => new Promise((reso
...
 });
-const runIsolated = async (config, cmd, options={}) => {
+const runIsolated = async (config, cmd, options = {}) => {
     const project = options.project || config.HYPERONE_PROJECT_MASTER;
     const config_dir = `/tmp/${Math.random()}/`;
     await mkDir(config_dir);
-    const envIsolate = {HYPERONE_CONFIG_PATH: config_dir};
-    await runProcess(`h1 login --username ${config.HYPERONE_USER} --password ${config.HYPERONE_PASSWORD}`, envIsolate);
+    const envIsolate = { HYPERONE_CONFIG_PATH: config_dir };
+    await runProcess(`h1 login -v --username ${config.HYPERONE_USER} --password ${config.HYPERONE_PASSWORD}`, envIsolate);
```
Comment (ad-m) [inline monitoring/run_monitoring.js]:
```suggestion
    await runProcess(`h1 login --username ${config.HYPERONE_USER} --password ${config.HYPERONE_PASSWORD}`, envIsolate);
```

File: monitoring/run_monitoring.js [hunk 5]
```diff
@@ -126,7 +150,68 @@ const sendMail = async (config, success, report) => {
...
+            result: 0,
+        });
+    }
+    console.log(`Failed to match line: ${line}`);
+};
+
+const runProcess = async (cmd, env = {}, options = {}) => new Promise((resolve, reject) => {
+    const timeout = options.timeout || 60 * 30;
+    const influxConn = options.influxConn;
+    console.log({ timeout, influxConn });
```
Comment (ad-m) [inline monitoring/run_monitoring.js]:
```suggestion

```

File: monitoring/run_monitoring.js [hunk 6]
```diff
@@ -126,7 +150,67 @@ const sendMail = async (config, success, report) => {
...
+        if ([' Timed out while running tests'].includes(help)) {
+            return;
+        }
+        return sendInflux(conn, 'testcase', {
+            title,
+            run: startTime,
+        }, {
+            result: -1,
+        });
+    } else if (line.includes(' ◌ ')) {
```
Comment (fredericosilva) [inline monitoring/run_monitoring.js]:
```suggestion
    }
    if (line.includes(' ◌ ')) {
```

**Pull request merged**